huaweicloud/services/acceptance/cloudtable/resource_huaweicloud_cloudtable_cluster_test.go

```diff
@@ -46,7 +46,7 @@ func TestAccCloudtableCluster_basic(t *testing.T) {
 						"data.huaweicloud_availability_zones.test", "names.0"),
 					resource.TestCheckResourceAttr(resourceName, "name", rName),
 					resource.TestCheckResourceAttr(resourceName, "storage_type", "ULTRAHIGH"),
-					resource.TestCheckResourceAttr(resourceName, "storage_size", "10240"),
+					resource.TestCheckResourceAttr(resourceName, "storage_size", "800"),
```

Comment:
it's better to use `TestCheckResourceAttrSet` as the value of `storage_size` is not a constant.